conftest.py

```diff
@@ -12,6 +13,9 @@ class TestFields(Enum):
     BUSINESS_NAME_TEST = 'hair_salon_inc'
 
 
+TestFields.__test__ = False
```

Comment:
What's this?

Reply:
Pytest thought that testfield because its name a test it self and gives us warning so i fixed it by telling it that its not

Reply:
ok